src/shared/helpers/filterMovies.ts

```diff
@@ -0,0 +1,15 @@
+import { Movie } from "../../models";
+
+export function filterMovies(
+  movies: Movie[],
+  searchingValue: string,
+  categories: string
+) {
+  return movies.filter(
+    (movie) =>
+      (categories === "ALL"
+        ? true
```

Comment:
make `categories` singular, and it is not good algorithm.
you always go through all list even if `category` is ALL and searchingValue is empty string;
Make it more effective.